openmrs-watcher/pom.xml

```diff
@@ -23,6 +23,12 @@
             <groupId>org.apache.camel.springboot</groupId>
             <artifactId>camel-debezium-mysql-starter</artifactId>
             <version>${camelVersion}</version>
+             <exclusions>
```

Comment:
Why is this exclusion required?

Reply:
Hi @wluyima, sorry I missed this comment. 

>Why is this exclusion required?

This is because it is a transitive dependency of `camel-debezium-mysql-starter`.

Reply:
Is spring-boot-starter already provided somewhere else?

Reply:
Thanks for for pointing out this.  It's not explicitly provided in this project's `pom`s except for [this line](https://github.com/openmrs/openmrs-eip/pull/18/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R27). I've explicitly included the dependency in the root pom.